internal/cnab/driver.go

```diff
@@ -109,6 +109,9 @@ func PrepareDriver(dockerCli command.Cli, bindMount BindMount, stdout io.Writer)
 	for env := range d.Config() {
 		driverCfg[env] = os.Getenv(env)
 	}
+	if driverCfg["CLEANUP_CONTAINERS"] != "true" && driverCfg["CLEANUP_CONTAINERS"] != "false" {
```

Comment:
I wonder if we really need this?

Reply:
I changed it to `os.LookupEnv` so that we don't rely on a specific key and only configure values that are set.